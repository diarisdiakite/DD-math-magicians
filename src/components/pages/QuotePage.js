import React from 'react';
import Quotes from '../childComponents/Quotes';

function QuotePage() {
  return (
    <div>
      <Quotes />
    </div>
  );
}

export default QuotePage;
