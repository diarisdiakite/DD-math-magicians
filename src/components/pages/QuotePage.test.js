import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import QuotePage from './QuotePage';

describe('Header component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <QuotePage />
        </MemoryRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
