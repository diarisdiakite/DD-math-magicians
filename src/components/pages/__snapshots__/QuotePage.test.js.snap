// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<div>
  <div
    className="main-container"
  />
</div>
`;
